src/middleware/cache.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// Note - private/public declarations will automatically be added as appropriate if not specified
+const presets = {
+	no: {
+		maxAge: 0,
+		conditions: 'no-cache, no-store, must-revalidate'
```

Comment:
There's an internal method `privatize()` used to add private. I figured it's easier to define everything without public/private and add one or the other as a final step rather than doing string replaces. Or something... can't quite remember the rationale, or whether I half evolved away from it, now